src/components/IdeaCardComponent.js

```diff
@@ -0,0 +1,48 @@
+import * as React from 'react';
+import Card from '@mui/material/Card';
+import CardContent from '@mui/material/CardContent';
+import CardMedia from '@mui/material/CardMedia';
+import Typography from '@mui/material/Typography';
+import { CardActionArea, CardActions } from '@mui/material';
+import ShowMoreButton from './Buttons/ShowMore';
+
+function IdeaCard(props) {
+    const { idea } = props;
+
+    if (!idea) return null;
+
+    function shortenDescription(idea) {
+        if (idea.description.length > 150) {
+            return idea.description.slice(0, 150) + '...';
+        }
```

Comment:
And if the idea is less than 150 characters - what should it return?

Reply:
It should return idea.description. Fixed.